sirepo/package_data/static/json/srw-schema.json

```diff
@@ -518,6 +518,7 @@
             "horizontalPosition": ["Horizontal Center Position [mm]", "Float"],
             "horizontalRange": ["Range of Horizontal Position [mm]", "Float"],
             "method": ["Single-Electron Spectrum Computation Method", "IntensityMethod"],
+            "mpiGroupCount": ["Number of MPI processes groups", "Integer", 2, "", 2],
```

Comment:
What is a group?

Reply:
That was based on this description I got from @moellep: "number of independent groups of MPI processes"

Reply:
There is a lot of text in SRW to choose from - these are all the same parameter:

 ['wm_nmm', 'i', 1, 'number of MPI Masters to use'],

:param _nmm: number of MPI "masters" to be used at 4D cross-spectral density (CSD) calculation

:param _n_mpi: number of independent "groups" of MPI processes (for 4D CSD calculation)

Reply:
Let's use this: `number of independent groups of MPI processes` 

MPI doesn't have a "process group" concept which was confusing me a bit. This wording is vaguer but understandable.